internal/helm/chart/builder_test.go

```diff
@@ -91,9 +91,8 @@ func TestRemoteReference_Validate(t *testing.T) {
 			wantErr: "invalid chart name 'iNvAlID-ChArT-NAmE!'",
 		},
 		{
-			name:    "ref with Artifactory specific invalid format",
-			ref:     RemoteReference{Name: "i-shall/not"},
-			wantErr: "invalid chart name 'i-shall/not'",
+			name: "ref with Artifactory specific valid format",
+			ref:  RemoteReference{Name: "i-shall/not"},
 		},
```

Comment:
Please add another two test cases for single `/` and for double+ `/` within the name.